### PR TITLE
Add documentation for GrainTracker ICs

### PR DESCRIPTION
**Pull request by permcody**
refs #10000

### CODE REVIEW

File: docs/content/documentation/systems/Postprocessors/phase_field/PolycrystalEBSD.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # PolycrystalEBSD
+
+PolycrystalEBSD uses an EBSDReader object and utilizes the algorithms in [Polycrystal Initial Conditions](/ICs/PolycrystalICs.md) to generate
+a reduced order parameter model from electron backscatter data. More information about this capability can be found [here]((ICs/EBSD.md).
```
Comment (aeslaughter) [inline docs/content/documentation/systems/Postprocessors/phase_field/PolycrystalEBSD.md]: I don't like using "here", also you seem to have an extra (.

 I would prefer something like:
```
...this capability is provided in the [EBSD](ICs/EBSD.md) object description.
```

File: docs/hidden.yml [hunk 2]
```diff
@@ -695,16 +695,10 @@ phase_field:
...
-    - /Postprocessors/FeatureFloodCount
     - /Postprocessors/FeatureVolumeFraction
     - /Postprocessors/GrainBoundaryArea
-    - /Postprocessors/GrainTracker
     - /Postprocessors/GrainTrackerElasticity
     - /Postprocessors/PFCElementEnergyIntegral
-    - /Postprocessors/PolycrystalCircles
-    - /Postprocessors/PolycrystalEBSD
-    - /Postprocessors/PolycrystalHex
-    - /Postprocessors/PolycrystalVoronoi
```
Comment (aeslaughter) [inline docs/hidden.yml]: It is really nice to see a few of these to start to disappear, thanks!

**Review by aeslaughter — CHANGES_REQUESTED** [review]
Looks good, just one small comment to consider.

**Comment by permcody** [comment]
Ready to review again @aeslaughter

**Review by aeslaughter — APPROVED** [review]

**Comment by aeslaughter** [comment]
@permcody If this passes I will do a squash and merge, which might be the better way to make these tiny doc fixes rather than go through the process of typing the change or pull in the content.

**Pull request merged**